rules/cloud/azure_network_virtual_device_modified_or_deleted.yml

```diff
@@ -0,0 +1,29 @@
+title: Azure Virtual Network Device Modified or Deleted
+id: 15ef3fac-f0f0-4dc4-ada0-660aa72980b3
+description: Identifies when a virtual network device is being modified or deleted. This can be a network interface, network virtual applicance, vitual hub, or virtual router.
+author: Austin Songer
+status: experimental
+date: 2021/08/08
+references:
+    - https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations
+logsource:
+  service: azure.activitylogs
+detection:
+    selection:
+        properties.message:
+            - MICROSOFT.NETWORK/NETWORKINTERFACES/WRITE
+            - MICROSOFT.NETWORK/NETWORKINTERFACES/JOIN/ACTION
+            - MICROSOFT.NETWORK/NETWORKINTERFACES/DELETE
```

Comment:
Not this ?
Microsoft.Network/networkInterfaces/tapConfigurations/write
Microsoft.Network/networkInterfaces/tapConfigurations/delete

Reply:
Nice. didn't see that.